src/components/organisms/news/news.html

```diff
@@ -0,0 +1,42 @@
+<section>
+  <div class="news-and-events-title">
+    <h2>News<span class="caption">お知らせ</span></h2>
+  </div>
+  <div class="news-contents">
+    <ul>
+      <li>
+        <time datetime="2018.06.20">2018.06.20</time>
+        <article>
+          ブックカフェ・コワーキング系サイト向けテーマ「Switch」をリリースしました。
+        </article>
```

Comment:
```suggestion
  <article>
        <time datetime="2018.06.20">2018.06.20</time>
        <h2>
          ブックカフェ・コワーキング系サイト向けテーマ「Switch」をリリースしました。
        </h2>
  </article>
```
article タグは記事セクションを囲む部分なので、↑のように書く方がいい感じです。
[
mdn ](https://developer.mozilla.org/ja/docs/Web/HTML/Element/article)の使用上の注意という項目には以下のようにあるので、見出しも設定しましょう。
> それぞれの <article> は、子要素として見出し (`[<h1>-<h6>]`(https://developer.mozilla.org/ja/docs/Web/HTML/Element/Heading_Elements) 要素) を含むなどの方法で識別できるようにするべきです。
https://developer.mozilla.org/ja/docs/Web/HTML/Element/article